scripts/run-pelorus-e2e-tests

```diff
@@ -0,0 +1,177 @@
+function ornds() { oc rollout status --namespace ${PELORUS_NAMESPACE} deployments "$@"; }
+function owpr() { oc wait pod --for=condition=Ready -n ${PELORUS_NAMESPACE} -l pelorus.konveyor.io/exporter-type="$*"; }
+set +a
+
+### Options
+OPTIND=1
+while getopts "h?b:v:" option; do
+    case "$option" in
+    h|\?) print_help;;
+    b)    demo_branch=$OPTARG;;
+    v)    venv_dir=$OPTARG;;
+    esac
+done
+
+if [ -z "${venv_dir}" ]; then
+    VENV="${DEFAULT_VENV}"
+else
+    VENV="${venv_dir}"
+fi
+
+if [ -z "${demo_branch}" ]; then
+    demo_branch="master"
+fi
+
+
+
+### MAIN
+# Create download directory
+DWN_DIR=$(TMPDIR="${VENV}" mktemp -d -t "${TMP_DIR_PREFIX}XXXXX") || exit 2
+
+echo "Temporary directory created: ${DWN_DIR}"
+
+# Cleanup download directory on exit
+trap 'cleanup_and_exit' INT TERM EXIT
+
+download_file_from_url "https://raw.githubusercontent.com/konveyor/mig-demo-apps/$demo_branch/apps/todolist-mongo-go/pelorus/values.yaml" "ci_values.yaml" "${DWN_DIR}"
+download_file_from_url "https://raw.githubusercontent.com/konveyor/mig-demo-apps/$demo_branch/apps/todolist-mongo-go/mongo-persistent.yaml" "mongo-persistent.yaml" "${DWN_DIR}"
+
+# Create namespace where pelorus and grafana, prometheus operators will get deployed 
+oc create namespace "${PELORUS_NAMESPACE}"
+
+# Modify downloaded files
+sed -i.bak 's/your_org/konveyor/g' "${DWN_DIR}/mongo-persistent.yaml"
+
+# Show what has been modified:
+diff -uNr "${DWN_DIR}/mongo-persistent.yaml" "${DWN_DIR}/mongo-persistent.yaml.bak"
+
+# From now on, exit if something goes wrong
+set -e
+
+# Ensure we are in the top-level directory of pelorus project
+pushd "${SCRIPT_DIR}/../"
+
+# Apply config maps for the exporters
+oc apply -f "charts/pelorus/configmaps"
+
+helm install operators charts/operators --namespace pelorus --debug --wait --wait-for-jobs
+
+# Wait for grafana and prometheus deployments to be rolled out
+retry 5m 1s ornds prometheus-operator
+retry 5m 1s ornds grafana-operator-controller-manager
+
+helm install pelorus charts/pelorus --namespace pelorus --debug --wait --wait-for-jobs
+
+# check final deployment
+retry 5m 5s ogns grafana-service
+retry 5m 5s ogns prometheus-operated
+retry 5m 5s ogns prometheus-pelorus
+
+# update exporter values and helm upgrade
+helm upgrade pelorus charts/pelorus --namespace pelorus --values "${DWN_DIR}/ci_values.yaml"
+
+retry 10m 5s owpr deploytime
+retry 10m 5s owpr committime
+
+oc create -f "${DWN_DIR}/mongo-persistent.yaml"
+
+retry 2m 5s oc wait pod --for=condition=Ready -n mongo-persistent -l app=mongo
+retry 10m 10s oc wait pod --for=condition=Ready -n mongo-persistent -l app=todolist
+
+# Test
+committime_route=$(oc get route -n ${PELORUS_NAMESPACE} committime-exporter -o=template='http://{{.spec.host | printf "%s\n"}}')
+deploytime_route=$(oc get route -n ${PELORUS_NAMESPACE} deploytime-exporter -o=template='http://{{.spec.host | printf "%s\n"}}')
+
+curl "$committime_route" 2>&1 | grep todolist
```

Comment:
hrm.. I have reservations about curl but it's the ideal.  Have we seen this run in prow?

Reply:
No, but I have ran it against real cluster and the only failure which I can think of is when the route is not accessible.

The script will exit with error when there is no todolist in the output of the prometheus exporter which is correct for both not accessible route or commit time pod not running properly.

Reply:
aye.. my local testing of curl to a cluster worked.. hopefully it will work in prow as well :)